rclcpp/src/rclcpp/subscription_base.cpp

```diff
@@ -354,3 +354,27 @@ SubscriptionBase::set_on_new_message_callback(
     throw_from_rcl_error(ret, "failed to set the on new message callback for subscription");
   }
 }
+
+#ifdef CONTENT_FILTER_ENABLE
+void
+SubscriptionBase::set_content_filter(
+  const std::string & filter_expression,
+  const std::vector<std::string> & expression_parameters)
+{
+  static_cast<void>(filter_expression);
+  static_cast<void>(expression_parameters);
+
+  rclcpp::exceptions::throw_from_rcl_error(RMW_RET_UNSUPPORTED, "content filter unsupported");
```

Comment:
Why this method is stub? we will enable CFT for user application in `rcl` and `rclcpp` so that user application and user defined topics can take advantage of CFT for next Humble Release. we already have consensus on this, see https://github.com/ros2/design/pull/282#issuecomment-1071170295